apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SessionPoolManager.cs

```diff
@@ -35,7 +37,13 @@ namespace Google.Cloud.Spanner.Data
     /// required.
     /// </summary>
     public sealed class SessionPoolManager
-    {
+    {        
+        // TODO: Should these be configurable?
```

Comment:
I wonder if there's a way for us to tie all of these to DI by taking advantage of the client builder.

At least offer a constructor for SessionPoolManager to receive a service provider etc.?

All of that of course can be done later.

Reply:
Yes, that would probably make sense - and can be pushed until post GA, probably.